database/migration/src/main/scala/cromwell/database/migration/liquibase/LiquibaseUtils.scala

```diff
@@ -11,6 +11,7 @@ import liquibase.{Contexts, LabelExpression, Liquibase}
 import org.hsqldb.persist.HsqlDatabaseProperties
 
 object LiquibaseUtils {
+  private val mutex = new Object
```

Comment:
OOC what is the difference between this and synchronizing on `LiquibaseUtils` itself ?

Reply:
See comment below.